rq/cli/helpers.py

```diff
@@ -48,12 +49,22 @@ def get_redis_from_config(settings, connection_class=Redis):
         sn = Sentinel(instances, socket_timeout=socket_timeout, password=password, db=db)
         return sn.master_for(master_name)
 
+    ssl = settings.get('REDIS_SSL', False)
+    if isinstance(ssl, string_types):
```

Comment:
Can we use `str` instead of `string_types`? We no longer need to support Python 2.